packages/lib/server/queries/teams/index.ts

```diff
@@ -101,8 +101,6 @@ export async function getTeamWithMembers(args: {
   if (userId) where.members = { some: { userId } };
   if (orgSlug) {
     where.parent = getSlugOrRequestedSlug(orgSlug);
-  } else {
-    where.parentId = null;
```

Comment:
This was preventing the onboarding members step in team creation wizard to load, since was setting `parentId` to null for a subteam, which has a `parentId`